docs/howtos/DataPlane Generation - DPG/03convenient.mdx

```diff
@@ -250,7 +300,37 @@ using Azure.ClientGenerator.Core;
 <TabItem value="csharp" label="CSharp" >
 
 ```csharp
-FIXME
+// If a model is input-only, there will be one public constructor with all required properties as parameter.
+// And the required properties without setter. Optional properties will have both getter and setter
+namespace Azure.AI.OpenAI.Models
+{
+  public partial class AzureCognitiveSearchIndexFieldMappingOptions
+  {
+    public AzureCognitiveSearchIndexFieldMappingOptions()
+
+    public string TitleFieldName { get; set; }
+    public string UrlFieldName { get; set; }
+    public string FilepathFieldName { get; set; }
+    public IList<string> ContentFieldNames { get; }
+    public string ContentFieldSeparator { get; set; }
+    public IList<string> VectorFieldNames { get; }
+    public IList<string> ImageVectorFieldNames { get; }
+  }
+}
+// If a model is output-only, there is no public constructor, and all properties only have getter, no setter.
+
+// If a model is roundtrip (input + output), there will be one public constructor with all required properties as parameter.
+// And all properties will have both getter and setter
```

Comment:
```suggestion
// All properties have both getter and setter
```

We should mention collection properties, because despite here we said "all properties have both getter and setter", the code below, `Data` property does not have a setter. Questions will pop up.

Reply:
added